File: Upcoder/Khối 11/Chương 2 Cấu trúc rẽ nhánh/Khach Sạn/KhachSan.py
# Nhập ngày đến và ngày đi
arrival, departure = map(int, input().split())

# Kiểm tra tính hợp lệ của ngày đến và ngày đi
if arrival < 1 or arrival > 7 or departure < 1 or departure > 7:
    print("NHAP SAI", end="")
else:
    # Tính số đêm
    total_nights = 0
    if departure >= arrival:
        total_nights = departure - arrival
    else:
        total_nights = (7 - arrival) + departure  # Tính toán qua cuối tuần

    # Tính tiền
    total_cost = 0
    for i in range(total_nights):
        current_day = (arrival + i - 1) % 7 + 1  # Lấy ngày hiện tại
        if current_day == 1:  # Chủ Nhật
            total_cost += 400000
        else:
            total_cost += 300000

    # Xuất ra số tiền phải trả
    print(total_cost, end="")

File: Upcoder/Khối 11/Chương 2 Cấu trúc rẽ nhánh/Khach Sạn/KhachSan.cs
using System;
namespace Khachsan
{
    public class Program
    {
        static void Main()
        {
            // Nhập ngày đến và ngày đi
            string[] input = Console.ReadLine().Split(' ');
            int arrival = int.Parse(input[0]);
            int departure = int.Parse(input[1]);
    
            // Kiểm tra tính hợp lệ của ngày đến và ngày đi
            if (arrival < 1 || arrival > 7 || departure < 1 || departure > 7)
            {
                Console.Write("NHAP SAI");
                return;
            }
    
            // Tính số đêm
            int totalNights = 0;
            if (departure >= arrival)
            {
                totalNights = departure - arrival;
            }
            else
            {
                totalNights = (7 - arrival) + departure; // Tính toán qua cuối tuần
            }
    
            // Tính tiền
            int totalCost = 0;
            for (int i = 0; i < totalNights; i++)
            {
                int currentDay = (arrival + i - 1) % 7 + 1; // Lấy ngày hiện tại
                if (currentDay == 1) // Chủ Nhật
                {
                    totalCost += 400000;
                }
                else
                {
                    totalCost += 300000;
                }
            }
            Console.Write(totalCost);
        }
    }
}
